### DISCUSSION TITLE
A1111 Checkpoint models not loading

### DISCUSSION THREAD

**Discussion by j-r-42** [comfyanonymous/ComfyUI#928] [Q&A]
Hi, I'm hoping somebody can help me out here. I just installed ComfyUI, no problems, everything seems to be running good, with one exception. I followed the directions for pointing ComfyUI to my A1111 checkpoint models, and it doesn't appear to locate them. Attached is a screen cap: top window has the directory where the checkpoint files are located; second window, the search paths I specified in the .yaml file; and the third, the result in ComfyUI showing "undefined" as far as checkpoints go.

Any ideas why this may be, or how I can about fixing it. I'm working on a MacBook Pro M2 Max 32GB. 
![models](https://github.com/comfyanonymous/ComfyUI/assets/44616571/eb47392c-2a99-4936-91e7-79a87d235410)

**Comment by comfyanonymous** [comment]
Add a / in front of Users in your base_path

**Reply by j-r-42** [reply]
Thank you!

I feel like an idiot ... I know better than that .. anyway, thanks again
